CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ if (WITH_PYTHON)
     find_package(Python3 COMPONENTS Interpreter Development)
 endif()
 
-if (MSVC)
+if (WIN32)
     # Use unsigned characters
     add_definitions(-J)
```

Comment:
These are compiler options, so isn't `MSVC` right?  Do other compilers support the MSVC options? or maybe this should be `MSVC or something`?